lambdas/ssm-parameters/src/ssm-parameters-handler.ts

```diff
@@ -0,0 +1,38 @@
+import { LambdaInterface } from "@aws-lambda-powertools/commons";
+import { getParametersByName } from "@aws-lambda-powertools/parameters/ssm";
+import { Parameter } from "@aws-sdk/client-ssm";
+
+const PARAMETER_TTL =
```

Comment:
Can you give time context to the TTL? It's not clear that it will default to 5mins (300 seconds). `CACHE_TTL_IN_SECONDS` might be better.